ttt.c

```diff
@@ -39,41 +43,92 @@ void draw_board(const char *t)
     printf("\n");
 }
 
+char check_row(const char *t)
```

Comment:
Since `check_row` and `check_col` have lines in common, can you introduce a helper function which accepts a flag to indicate wether row or column should be checked?